### ISSUE TITLE
Spurious message: "Error: Loading article (Url: https://upload.wikimedia.org/wikipedia/commons/6/65/Lock-green.svg"

### ISSUE THREAD

**Issue by tigran123** [kiwix/kiwix-android#3647]
**Describe the bug**

When loading any WikiPedia English article from the latest 102GB ZIM file an annoying popup window with the following error message appears:

"Error: Loading article (Url: https://upload.wikimedia.org/wikipedia/commons/6/65/Lock-green.svg"

Actually, there are two or three messages like the above appear, I am just showing the typical one.

But this does not prevent displaying the actual article. All data inside article (images, formulae etc) appear to be ok.

**Expected behavior**

No error message.

**Steps to reproduce the behavior:**
1. Make sure Internet is turned off (the whole point of Kiwix is to access data where no Internet is available)
2. Open any article in Wikipedia (e.g. "Wigner quasiprobability distribution" -- I always use it for testing as it has images, formulae and even animated gifs.)
3. Observe the error message at the bottom.

**Screenshots**

**Environment**
- Version of Kiwix Android :  3.9.1 from https://download.kiwix.org/release/kiwix-android/kiwix-3.9.1.apk
- Device :  Samsung Galaxy Tab S4 and Note10+
- OS version :  Android 10 and 12

**Logs**

**Comment by tigran123** [comment]
The same happens when loading articles from the Russian WikiPedia, but the error messages are different, e.g. I get this:

"Error: Loading article (Url: https://upload.wikimedia.org/wikipedia/commons/2/23/Icons-mini-file_acrobat.gif"

**Comment by Jaifroid** [comment]
Thanks for the report. This is a very longstanding issue and is not limited to Android - you can also see these errors in the Browser Extension console log. The issue is with mwOffliner not rewriting those links. The errors are harmless and just indicate that those items have been blocked by the webview or the browser.

**Comment by Jaifroid** [comment]
Closing as duplicate of https://github.com/openzim/mwoffliner/issues/1280.

**Comment by CaptainLexington** [comment]
The error message is harmless but extremely annoying. Several display every time I open an article. imo the issue is not a duplicate upstream because I don't care whether the images are included or not, or whether the attempt to request them has failed. I just want a way to prevent the toast notifications from blocking the reading window. Perhaps an option to prevent making http requests that will definitely fail, i.e. if the device is offline.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Last remark from @CaptainLexington makes sense. Do we have a way that this toast message appears one per article load an disappears after a while?

**Comment by tigran123** [comment]
Yes, those messages are extremely annoying and it is very difficult to get used to them. One has to almost learn to exercise spatio-temporal "loss of consciousness" -- temporal for the duration of the message and spatial for the area of space it occupies -- both need to be ignored with the power of will :)

**Comment by kelson42** [comment]
@tigran123 We will implement your request, sorry for the delay and thank you for your patience. Moving to the to be `3.10.0` to be published very soon.
